scripts/install_third_party_libs.py

```diff
@@ -210,6 +219,74 @@ def ensure_system_python_libraries_are_installed(package, version):
     install_backend_python_libs.pip_install_to_system(package, version)
 
 
+def rewrite_android_proto_files():
+    """Edit all android proto files."""
+    # Since there is no simple configuration for imports when using protobuf to
+    # generate Python files we need to manually fix the imports.
+    # See: https://github.com/protocolbuffers/protobuf/issues/1491
+    protobuf_dir = (
+        pathlib.Path(
+            os.path.join(
+                common.THIRD_PARTY_DIR,
+                'oppia-proto-api-8f3cde883c31785438e80656a5b6bb26bd01b6a1'))
+            .glob('**/*.proto'))
+    for p in protobuf_dir:
+        if p.suffix == '.proto':
+            # Remove package statement.
+            # Example: 'package org.oppia.proto.v1.api'.
+            common.inplace_replace_file(
+                p.absolute(),
+                r'^package ([^\s]+)',
+                r'')
+            # Remove option statement.
+            # Example: 'option java_package = "org.oppia.proto.v1.api";'.
+            common.inplace_replace_file(
+                p.absolute(),
+                r'^option java_.+',
+                r'')
+            # Update import statement.
+            # Example: 'import "org/oppia/proto/v1/api/state.proto";' to
+            # 'import "state.proto";'.
+            common.inplace_replace_file(
+                p.absolute(),
+                r'^import (?!\"google)([^\s]+)[\\/]([^\s]+)',
+                r'import "\2')
+            # Remove all subpackage directories.
+            # Example: 'org.oppia.'.
+            common.inplace_replace_file(
+                p.absolute(),
+                r'org.oppia.*\.',
+                r'')
+
+
+def move_all_proto_files_to_third_party():
+    """Move all proto files from subdirectories
+    to the third_party folder.
+    """
+    oppia_proto_api_path = (
+        os.path.join(
+            common.THIRD_PARTY_DIR,
+            'oppia-proto-api-8f3cde883c31785438e80656a5b6bb26bd01b6a1'))
+    protobuf_dir = (
+        pathlib.Path(
+            oppia_proto_api_path).glob('**/*.proto'))
+
+    for p in protobuf_dir:
```

Comment:
Please choose a more descriptive name than 'p'.

Reply:
Updated with the `file` as we are getting file from the `protobuf_dir `.